{"name": "test_add_user_already_exists", "status": "passed", "steps": [{"name": "add user {'user_name': 'adim333', 'user_password': 'x0z3EvGC'}", "status": "passed", "start": 1679895624268, "stop": 1679895624318}, {"name": "getting the user adim333", "status": "passed", "start": 1679895624318, "stop": 1679895624388}, {"name": "add user {'user_name': 'adim333', 'user_password': 'x0z3EvGC'}", "status": "passed", "start": 1679895624388, "stop": 1679895624445}], "attachments": [{"name": "log", "source": "a81bb5b3-dd9c-483d-97fe-ff2ad4cf4eff-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "4971db69-661c-4ee2-9d5a-1370d7ede692-attachment.txt", "type": "text/plain"}], "start": 1679895624268, "stop": 1679895624445, "uuid": "58201ef6-3f22-466f-9fad-3cbd60275f59", "historyId": "d1547a7b0f7584a5cc7d5fc11e478d70", "testCaseId": "2653eaebae43387b15ebaa804ec6db84", "fullName": "tests.test_users.TestUsers#test_add_user_already_exists", "labels": [{"name": "story", "value": "a client cant add exists usernames"}, {"name": "epic", "value": "users api"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup_teardown')"}, {"name": "tag", "value": "users"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_users"}, {"name": "subSuite", "value": "TestUsers"}, {"name": "host", "value": "LAP-adim"}, {"name": "thread", "value": "16840-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_users"}]}